1-exercises/B-while-loop/exercise.js

```diff
@@ -5,8 +5,39 @@
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
 
+// function evenNumbers(n) {
+//     // TODO
+//     let start = 0;
+//     let result = []; 
+//     if (n < 10) {
+//         while (start < n) {
+//             result.push(start);
+//             start+=2;
+//         } 
+//     } else if (n >= 10){
+       
+//             while (start < n * 2) {
+//                 result.push(start);
+//                 start+=2;
+//             } 
+       
+//     }
+    
+//     console.log(result);
+//     return result;
+//}
+
 function evenNumbers(n) {
     // TODO
+    let start = 0;
+    let result = [];
+
+    while (result.length < n) {
+        result.push(start);
+        start+=2;
+    }
+
+    return console.log(result);
```

Comment:
```
function evenNumbers(n) {
  let counter = 0;
  let evenNum = 0;
  while (counter < n) {
    console.log(evenNum);
    evenNum += 2
    counter++;
  }
}
```
The above prints every value on a different line. As an exercise let do this to print them as a comma separated string:
1. Keep track of all the even numbers in the loop (hint: use array)
2. After loop, console.log the array using the `join`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join